modules/openapi-generator/src/main/resources/php-slim4-server/openapi_data_mocker.mustache

```diff
@@ -127,6 +127,22 @@ final class OpenApiDataMocker implements IMocker
         $exclusiveMinimum = false,
         $exclusiveMaximum = false
     ) {
+        $dataFormat = is_string($dataFormat) ? strtolower($dataFormat) : $dataFormat;
+        switch ($dataFormat) {
+            case IMocker::DATA_FORMAT_INT32:
+                // -2147483647..2147483647
+                $minimum = is_numeric($minimum) ? max($minimum, -2147483647) : -2147483647;
+                $maximum = is_numeric($maximum) ? min($maximum, 2147483647) : 2147483647;
+                break;
+            case IMocker::DATA_FORMAT_INT64:
+                // -9223372036854775807..9223372036854775807
+                $minimum = is_numeric($minimum) ? max($minimum, -9223372036854775807) : -9223372036854775807;
+                $maximum = is_numeric($maximum) ? min($maximum, 9223372036854775807) : 9223372036854775807;
+                break;
+            default:
+                // do nothing, unsupported format
```

Comment:
What about throwing a warning/error saying it's not supported so the users get notified?

Reply:
Accordingly to OAS spec any value can be used in `format` field. So it's not an exception to me. If this was a part of standalone library\package I would add throwing an exception, but it's autogenerated build, so nobody will catch exceptions in 99% cases.

However, PHP is able to raise warning or notice which doesn't stop code execution, but breaks JSON output.

Reply:
Ok. That's fine with me.